vllm_ascend/platform.py

```diff
@@ -46,8 +43,6 @@
     VllmConfig = None
     FlexibleArgumentParser = None
 
-os.environ["RAY_EXPERIMENTAL_NOSET_ASCEND_RT_VISIBLE_DEVICES"] = "1"
```

Comment:
I don't think this can be remove.
```
os.environ["RAY_EXPERIMENTAL_NOSET_ASCEND_RT_VISIBLE_DEVICES"] = envs.RAY_EXPERIMENTAL_NOSET_ASCEND_RT_VISIBLE_DEVICES
```